common/libs/units.ts

```diff
@@ -103,7 +104,17 @@ const fromTokenBase = (value: TokenValue, decimal: number) =>
 const toTokenBase = (value: string, decimal: number) =>
   TokenValue(convertedToBaseUnit(value, decimal));
 
-const isEtherUnit = (unit: string) => unit === 'ether' || unit === 'ETH';
+const isNetworkUnit = (
+  unit: string,
+  networks: AppState['config']['networks'],
+  nodes: AppState['config']['nodes']
+) => {
+  const selectedNode =
+    nodes.staticNodes[nodes.selectedNode.nodeId] || nodes.customNodes[nodes.selectedNode.nodeId];
+  const selectedNetwork =
+    networks.staticNetworks[selectedNode.network] || networks.customNetworks[selectedNode.network];
+  return unit === selectedNetwork.unit;
+};
```

Comment:
This seems like overkill, passing all of networks and nodes state. Shouldn't we just pass the current network? That's much less data for each component to have as a prop, they only need to pass the result of `getSelectedNetwork`.

Reply:
Oh I also see you made a `getNetworkUnit` selector, maybe even just that.

Reply:
yea actually that'd be a lot easier, forgot I even made a selector for that